packages/synapse-react-client/src/components/Evaluation/EvaluationRoundEditor.tsx

```diff
@@ -257,28 +268,46 @@ export const EvaluationRoundEditor: React.FunctionComponent<
 
             <Row>
               <Col>
-                <CalendarWithIconFormGroup
-                  value={startDate}
-                  setterCallback={setStartDate}
+                <DateTimePicker
                   label="Round Start"
-                  isValidDate={disallowDatesBeforeNow}
-                  disabled={dayjs().isSameOrAfter(
-                    evaluationRoundInput.roundStart,
-                  )}
+                  value={startDate}
+                  onChange={setStartDate}
+                  onError={error => setStartDateError(error)}
+                  disabled={disableStartInput}
+                  // Don't allow the value to be a past date only if the input is not disabled.
+                  // i.e. allow a past date if the round has already started
+                  disablePast={!disableStartInput}
+                  slotProps={{
+                    textField: {
+                      helperText: startDateError
+                        ? 'Start date cannot be in the past'
```

Comment:
Noticed this hard coded error message.  Is this the only error that can happen with this start date selector?

Reply:
Just realized that this also catches the invalid date case. Will replace with a message translator